tests/playTests.sh

```diff
@@ -165,6 +165,22 @@ rm tmp
 $ZSTD -f tmp && die "tmp not present : should have failed"
 test ! -f tmp.zst  # tmp.zst should not be created
 
+$ECHO "test : compress multiple files"
+echo hello > tmp1
+echo world > tmp2
+$ZSTD tmp1 tmp2 -o "$INTOVOID"
```

Comment:
Is this test supposed to work ?
I was expecting it to fail, because there are 2 input files associated with `-o` command.

Reply:
It only works with `/dev/null` or `-c`.